linkis-commons/linkis-storage/src/main/scala/org/apache/linkis/storage/utils/FileSystemUtils.scala

```diff
@@ -113,4 +114,15 @@ object FileSystemUtils extends Logging {
     true
   }
 
+  /**
+   * 返回文件总行数
+   * @param fileSource
+   * @return
+   *   TotalLine
+   */
+  def getTotalLine(fileSource: FileSource): Int = {
+    fileSource.collect()
```

Comment:
optimiz to FileSystemUtils.getTotalLine(fsPath, fileSystem))

Reply:
done